config/power_virtual_servers_regions.yml

```diff
@@ -0,0 +1,44 @@
+dal:
```

Comment:
Since _both_ `us-south` and `dal` are available in the IBM Cloud service catalog we need to include them both on the list here.